extensions/wikia/SpecialDiscussionsLog/templates/SpecialDiscussionsLog_inputForm.mustache

```diff
@@ -1,5 +1,11 @@
-<form method="get">
+<form method="get" class="form-username">
 	<label for="username">{{userNameLabel}}</label>
 	<input id="username" name="username" type="text" value="{{userName}}" required>
 	<input type="submit" value="{{viewLogsAction}}">
 </form>
+<div class="form-divider">{{orLabel}}</div>
+<form method="get" class="form-ipaddress">
+	<label for="ipaddress">{{ipAddressLabel}}</label>
+	<input id="ipaddress" name="ipaddress" type="text" value="{{ipAddress}}" required>
```

Comment:
Hey, I think we're using lower camelcase for ID's: https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#class-and-id-naming-conventions